### ISSUE TITLE
The system cannot find the file specified. Please try again later.

### ISSUE THREAD

**Issue by LordMinion777** [ClusterM/hakchi2#15]
So I got some .NES files (10) and I checked them all except "Original 30 games". Then I select "synchronize selected games whit NES mini" and I HOLD the reset button and then press the power button. It said the games are on. I go plug it to my TV and I only had those 10 games that I installed and I didnt have the original ones. Then I try it again and this time I tick the "Original 30 games" box and then "synchronize selected games" but a window pops up and says: The system cannot find the file specified. Please try again later.
Is my NES broken? Any help would be appriciated.

**Comment by brandonlbell** [comment]
I am in the exact same boat. Was able to load ROMs yesterday - no problem. When I went in to try and add them again today, this is the same area I see.

**Comment by ClusterM** [comment]
Please try version 2.09. If you still have problems please contact me: clusterrr@clusterrr.com

**Comment by GoodbyeWorld** [comment]
I had the same issue.  I ended up just starting fresh.

**Comment by ClusterM** [comment]
GoodbyeWorld, try to download debug version, run it. If you still has this problem archive the whole hakchi2 directory and send it to me (it should contain temp folder and debuglog.txt).

**Comment by GoodbyeWorld** [comment]
will do - I encountered this after upgrading from 2.08 to 2.09 by replacing all contents with the new version.

In case it helps, the files it complained were missing were game rom names like "CLV-AZSR-SOMETHING" or something like like. 

I also noticed that with 2.09, while it came with a database of rom details, it didn't update roms I'd already added, so I was OK with just starting fresh with 2.09 and re-adding them all.

I'll try to reproduce this with the debug version and will post back if I'm able to.
